pkg/scraping/scraper.go

```diff
@@ -109,11 +109,20 @@ func (s *Scraper) queryNode(ctx context.Context, url string, events chan entitie
 		return
 	}
 	events <- entities.NewHeightEvent(url, ts, v, h)
+
+	bs, err := node.baseTarget(ctx, h)
+	if err != nil {
+		events <- entities.NewBaseTargetEvent(url, ts, v, h, bs)
```

Comment:
`bs` value shouldn't be used in case of error
Send `BaseTargetEvent` if no error occurred.